lib/dns-round-robin.coffee

```diff
@@ -0,0 +1,43 @@
+dns = require('dns')
+URL = require('url')
+http = require('https')
+addresses=[]
+
+###
+# have set default time out for http.get to 100 milliseconds
+#
+# resolve_ip function resolves a domain name to all the corresponding host and try hitting 
+#   one by one until a response is got from the host (this response can even be a 404 or 400).
+#   
+# Example usage:
+#   resolve_ip 'itunes.com', (err, res) ->
+#     console.log res
+#     return
+###
+
+class ResolveIP
+  @resolve_ip = (hostName, callback, timeout=100) ->
```

Comment:
Please move this out to a config like rod did in this PR: https://github.com/intellinote/inote-util/pull/7/files We may need longer than 100 ms on prod